resources/views/welcome.blade.php

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
I believe all these should be in the layout template instead right? @m-triassi

Reply:
Actually i don't think this view even gets used anymore. iirc it was deleted in some other pull request, and this is restoring it for some reason.